updaterelease.sh

```diff
@@ -358,6 +358,32 @@ update_config_yml() {
   fi
 }
 
+# Generates Javadoc shortlinks for the current snapshot release.
+generate_snapshot_javadoc_shortlinks() {
+  # Creates 2 sub-folders for each non-nested class inside javadocshortcuts.
+  # This ensures that each class documentation is accessible using  https://guava.dev/ClassName
+  # or https://guava.dev/classname.
+  if [[ "$RELEASE" != "snapshot" ]]; then
+    return
+  fi
+  for F in $(find releases/snapshot/api/docs/com/google/common -name '[A-Z]*.html' -not -path '*/class-use/*' -not -name '*.*.*'); do
```

Comment:
```suggestion
  for F in $(find releases/snapshot-jre/api/docs/com/google/common -name '[A-Z]*.html' -not -path '*/class-use/*' -not -name '*.*.*'); do
```

Reply:
(What you have should work, but we generally link to `snapshot-jre` nowadays for consistency.)